pyproject.toml

```diff
@@ -6,6 +6,7 @@ license = "BSD-2-Clause"
 authors = ["runZero <support@runzero.com>"]
 readme = "README.md"
 homepage = "https://runzero.com/"
+documentation = "https://runzeroinc.github.io/runzero-sdk-py"
```

Comment:
We are missing the version bump above this.

Reply:
Not yet, that comes in the next PR!